transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_account_update.sql

```diff
@@ -43,43 +43,52 @@ WITH customers_with_cloud_paid_subs as (
     customers_with_cloud_paid_subs.hightouch_sync_eligible
     AND customers_with_cloud_paid_subs.status in ('canceled', 'active') 
     AND customers_with_cloud_paid_subs.SKU = 'Cloud Professional'
+), customer_to_account as (
+    SELECT
+        -- Data for debugging
+        email,
+        dwh_id_account_id,
+        dwh_id_account_external_id,
+        contact_account_id,
+        contact_account_external_id,
+        lead_account_id,
+        lead_account_external_id,
+        domain_account_id,
+        domain_account_external_id,
+        -- Data to be updated
+        -- Select account id based on priority:
+        --  1. external id match
+        --  2. account from contact
+        --  3. account from lead
+        --  4. account with matching domain
+        case
+            when has_external_id_match then 'existing external id'
```

Comment:
@ifoukarakis - do you know where `has_external_id_match` is defined?

Reply:
Nice catch. Seems this was an old name and I copy-pasted from some old part of the code. Fixing it in hotfix PR.

Reply:
wondering if there is a tool out-there to catch these kind of simple referential errors.

Reply:
Best option right now seems to be DBT's slim CI